src/store/actor.ts

```diff
@@ -57,7 +57,12 @@ export const useActor = defineStore("actor", {
 			return this.agent.getBrowser();
 		},
 		avifSupported(): boolean {
-			return this.browser.name === "Chrome" && parseInt(this.browser.version as string, 10) >= 100;
+			const version = parseInt(this.browser.version as string, 10);
+			switch(this.browser.name) {
+				case "Chrome":  return version >= 85;
```

Comment:
The minimal AVIF support version for Chrome is >=100. Any lower version suffers from a [major rendering issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1274200), effectively reducing support to zero.

Reply:
Ah, interesting, I didn't pick that up. I better change it back to 100 then. :)